### DISCUSSION TITLE
Lesson 6 : Ganache Transactions not showing  (MacOS Catalina 10.15.7)

### DISCUSSION THREAD

**Discussion by JamiDEV** [smartcontractkit/full-blockchain-solidity-course-py#438] [Q&A] [answered]
Perhaps this isn't a bad result but I'm curious as to why I'm not seeing any transactions in Ganache when attempting to deploy to a persistent Ganache?

here's my contract deploying, I don't see any local RPC client IP
![Screen Shot 2021-11-24 at 9 17 23 PM](https://user-images.githubusercontent.com/53233466/143379145-3406ba5a-b371-4d4a-a3b6-d283a55998e8.png)

Heres the Ganache transactions section, no transactions. 
![Screen Shot 2021-11-24 at 9 20 34 PM](https://user-images.githubusercontent.com/53233466/143379217-166f6efb-5e32-4009-9d27-ff6cd442957e.png)

should I not be worried?

**Comment by cromewar** [comment, marked as answer]
Hello @JamiDEV Actually it meas you are using 2 different ganaches, the one running on your terminal which is `ganache-cli` and `ganache-ui` so if you want to use ganache ui, you should run `brownie run script --network ganache-ui` being this command whatever to have configured before, if you did the same as the course it should be `ganache-local`.

best regards, Cromewar.

**Reply by JamiDEV** [reply]
I get an error when I attempt the " brownie run script", I then tried "brownie run scripts" and still got an error see below. 
![Screen Shot 2021-11-27 at 10 13 46 PM](https://user-images.githubusercontent.com/53233466/143730563-c07e477d-d126-4669-a6e7-45bf5e5a104d.png)

**Reply by cromewar** [reply]
Hello @JamiDEV I wrote `ganache-ui` just as example, the network ID may vary depending how you configure it, on the course Patrick set the ganache UI as `ganache-local` but again the ID (name) is up to you. So try `brownie networks list` and see how you have named the network.

Also remember you can always add a new one with `brownie networks add`, use the [brownie documentation](https://eth-brownie.readthedocs.io/en/stable/network-management.html) as reference.
